packages/create-next-app/templates/api/js/next.config.mjs

```diff
@@ -0,0 +1,16 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
```

Comment:
Can we use `next.config.ts` please?

Reply:
This one's a `js` template, have `next.config.ts` on the `ts` one!